tests/sqllogictests/suites/query/02_function/02_0012_function_datetimes

```diff
@@ -112,6 +116,11 @@ select to_datetime('9999-12-31 23:59:')
 ----
 9999-12-31 23:59:00.000000
 
+query T
+select to_unix_timestamp('2023-04-06 04:06:23.231808')
```

Comment:
Better to add a timezone to test different timezone is expected:

like to_datetime:

query T
select to_datetime('2022-12-31T23:59:59+00:00')
----
2022-12-31 23:59:59.000000

In this test file, first set timezone='UTC', so we need to add a test with  

to_unix_timestamp(2022-12-31T23:59:59+00:00) and to_unix_timestamp(2022-12-31T23:59:59+08:00) check the result is right, and the diff needs to be 8*60*60 seconds.